Composer/packages/server/src/constants.ts

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+import { QnABotTemplateId } from "@bfc/shared";
```

Comment:
nit: this should be single quoted, and should have been formatted pre-commit by prettier. Do you know why it's not?

Reply:
Formatter fixed, thanks for the help!